skeleton/migrations/install_user_schema.php.tpl

```diff
@@ -22,7 +22,7 @@ class install_user_schema extends \phpbb\db\migration\migration
 
 	static public function depends_on()
 	{
-		return array('\{EXTENSION.vendor_name}\{EXTENSION.extension_name}\migrations\install_module');
+		return array('\phpbb\db\migration\data\v31x\v314');
```

Comment:
A depends on is always needed (especially when making schema changes), or unit testing could fail